src/Thujohn/Twitter/TwitterServiceProvider.php

```diff
@@ -62,4 +62,4 @@ public function provides()
 		return ['ttwitter'];
```

Comment:
Yup, but changing the facade to:

``` php
protected static function getFacadeAccessor() { return '\Thujohn\Twitter\Twitter'; }
``` 
Produced a different error:

```
Exception: Undefined index: ttwitter::config
```

I'll keep it how it's setup in the PR as it's working fine in my project.